packages/flutter/test/widgets/autocomplete_test.dart

```diff
@@ -421,6 +421,130 @@ void main() {
     expect(textEditingController.text, lastOptions.elementAt(0));
   });
 
+  group('optionsViewOpenDirection', () {
+    testWidgets('unset (default behavior): open downward', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(
+            body: Center(
+              child: RawAutocomplete<String>(
+                optionsBuilder: (_) => <String>['a'],
+                fieldViewBuilder: (_, TextEditingController controller, FocusNode focusNode, __) {
+                  return TextField(controller: controller, focusNode: focusNode);
+                },
+                optionsViewBuilder: (_, __, ___) => const Placeholder())))));
+      await tester.showKeyboard(find.byType(TextField));
+      final Offset fieldBottomLeft = tester.getBottomLeft(find.byType(TextField));
+      final Offset optionsViewTopLeft = tester.getTopLeft(find.byType(Placeholder));
+      final Offset optionsViewCenter = tester.getCenter(find.byType(Placeholder));
+      expect(fieldBottomLeft, offsetMoreOrLessEquals(optionsViewTopLeft));
+      expect(optionsViewCenter.dy, greaterThan(optionsViewTopLeft.dy));
+    });
+
+    testWidgets('down: open downward', (WidgetTester tester) async {
+      await tester.pumpWidget(
+          MaterialApp(
+            home: Scaffold(
+              body: Center(
+                child: RawAutocomplete<String>(
+                  optionsViewOpenDirection: OptionsViewOpenDirection.down, // ignore: avoid_redundant_argument_values
+                  optionsBuilder: (_) => <String>['a'],
+                  fieldViewBuilder: (_, TextEditingController controller, FocusNode focusNode, __) {
+                    return TextField(controller: controller, focusNode: focusNode);
+                  },
+                  optionsViewBuilder: (_, __, ___) => const Placeholder())))));
+      await tester.showKeyboard(find.byType(TextField));
+      final Offset fieldBottomLeft = tester.getBottomLeft(find.byType(TextField));
+      final Offset optionsViewTopLeft = tester.getTopLeft(find.byType(Placeholder));
+      final Offset optionsViewCenter = tester.getCenter(find.byType(Placeholder));
+      expect(fieldBottomLeft, offsetMoreOrLessEquals(optionsViewTopLeft));
```

Comment:
This test case is possibly a bit easier to think about if the options view looks a little more like a real options view would, showing the value of an option.

Also perhaps inline the finder calls into the expectations, since they're pretty concise and self-explanatory:
```suggestion
                  optionsViewBuilder: (_, __, ___) => const Text('a'))))));
      await tester.showKeyboard(find.byType(TextField));
      expect(tester.getBottomLeft(find.byType(TextField)),
        offsetMoreOrLessEquals(tester.getTopLeft(find.text('a'))));
```